wgsl/index.bs

```diff
@@ -1960,12 +1960,12 @@ declaration of the identifier as a type alias or structure type.
   </xmp>
 </div>
 
-# Variable and const # {#variables}
+# `var` and `let` # {#variables}
```

Comment:
What if this was `Variables and Constants` and then we say a constant declaration uses `let` instead of calling it a let declaration in the spec? It just reads a bit strange to me when the spec says a let ...

Reply:
The widespread usage of 'constant' and friends in the spec is a little weird given this change, but I think we'll be able to chip away at the more awkward of the wordings, and hopefully it won't be so obvious that 'constant' is vestigial to `const`.

Reply:
We can fix things like this in editorial updates.   I don't think this should block this PR.